homeassistant/components/airtouch5/config_flow.py

```diff
@@ -0,0 +1,65 @@
+"""Config flow for Airtouch 5 integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from airtouch5py.airtouch5_simple_client import Airtouch5SimpleClient
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema({vol.Required(CONF_HOST): str})
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> None:
+    """Validate the user input allows us to connect.
+
+    Data has the keys from STEP_USER_DATA_SCHEMA with values provided by the user.
+    """
+
+    client = Airtouch5SimpleClient(data[CONF_HOST])
+
+    try:
+        await client.test_connection()
+    except Exception:
+        raise CannotConnect
+
+
+class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for Airtouch 5."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle the initial step."""
+        errors: dict[str, str] | None = None
+        if user_input is not None:
+            try:
+                await validate_input(self.hass, user_input)
+            except CannotConnect:
```

Comment:
```suggestion
                await client.test_connection()
            except Exception:
```

Maybe you want to log the exception to give the user hints in the logs as to why the connection failed

Reply:
There is no authentication in the protocol, we should only fail if the IP is wrong.